packages/select/src/index.ts

```diff
@@ -1,2 +1,7 @@
-export { default, useListbox, UseListboxProps, useMenuRef } from './react/index'
+import '@pluralsight/ps-design-system-icon/styles.css'
+// import '@pluralsight/ps-design-system-menu/styles.css'
```

Comment:
@jaketrent So I ran storybook in Select with this uncommented and it broke. When I comment them out it works and icons look correct. I thought we were suppose to do this. I can't figure out what is going on with this.  I know menu also imports `icon/styles.css`

Reply:
Yes, we are supposed to include all the dependant component styles. This is for user convenience, so they only have to import the top-level component /styles.css.